index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.3/css/bulma.min.css">
+    <script
+      src="https://code.jquery.com/jquery-3.5.1.min.js"
+      integrity="sha256-9/aliU8dGd2tb6OSsuzixeV4y/faTqgFtohetphbbj0="
+      crossorigin="anonymous"
+    ></script>
+    <title>NASA App</title>
+</head>
+<body>
+    <h1>SpaceX LaunchPad</h1>
+    <nav class="static">
+        <div class="btn-group">
+            <button>Past</button>
+            <button>Current</button>
+            <button>Future</button>
+        </div>
+    </nav>
+  <main>
+    <h2>Count Down To Launch: </h2>
```

Comment:
We need 3 separate div containers here, one for past, one for current, one for next